index.bs

```diff
@@ -10722,6 +10654,64 @@ for the specific requirements that the use of
 </div>
 
 
+<h4 oldids="dom-unrestricteddouble dom-unrestrictedfloat idl-unrestricted-double idl-unrestricted-float"
+    id="Unrestricted" extended-attribute lt="Unrestricted">[Unrestricted]</h5>
+
+If the [{{Unrestricted}}] [=extended attribute=] appears on one of the [=float types=],
+it creates a new IDL type such that that when an ECMAScript Number is converted to the IDL type,
+IEEE 754 non-finite, and special "not a number" values (NaNs) are allowed. [[!IEEE-754]]
+
+The [{{Unrestricted}}]
+extended attribute must
+[=takes no arguments|take no arguments=].
+
+A type annotated with the [{{Unrestricted}}] extended attribute must not appear
+in a [=read only=] attribute. A type that is not a [=float type=] must not
```

Comment:
Is there any reason we don't allow `[Unrestricted]` appeared in a read only attribute? There is such usage in current spec, like https://html.spec.whatwg.org/multipage/media.html#htmlmediaelement

Reply:
Because the conversion from `float` and `double` to an **ECMAScript** `number` doesn’t require it.

Reply:
But the prose says that only if the extended attribute is specified it can contain the additional values.

Reply:
Specifically the HTMLMediaElement case returns NaN which requires Unrestricted.